docs/spark-getting-started.md

```diff
@@ -32,7 +32,7 @@ The latest version of Iceberg is [{{% icebergVersion %}}](../../../releases).
 
 Spark is currently the most feature-rich compute engine for Iceberg operations. 
 We recommend you to get started with Spark to understand Iceberg concepts and features with examples.
-You can also view documentations of using Iceberg with other compute engine under the **Engines** tab.
+You can also view documentations of using Iceberg with other compute engine.
```

Comment:
instead of removing this, we should rather update it to "...with other compute engines under the [Multi-Engine Support](https://iceberg.apache.org/multi-engine-support/) page"